Makefile

```diff
@@ -4,47 +4,60 @@
 CCFLAGS = -Wall -c
 
 ifeq ($(OS),Windows_NT)
-	os = windows
-	CCFLAGS += -D WIN32 -D IA32
-	LDFLAGS += -l rasapi32 -l wininet -Wl,--subsystem,windows
-	BIN = binaries/windows/pac
+    os = windows
+    CCFLAGS += -D WIN32
+    # 32 bit `make` utility over 64 bit OS
+    ifeq ($(PROCESSOR_ARCHITEW6432),AMD64)
+        CCFLAGS += -D AMD64
+        BIN = binaries/windows/pac_amd64
+    else
+        ifeq ($(PROCESSOR_ARCHITECTURE),AMD64)
+            CCFLAGS += -D AMD64
+            BIN = binaries/windows/pac_amd64
+        endif
+        ifeq ($(PROCESSOR_ARCHITECTURE),x86)
+            CCFLAGS += -D IA32
+            BIN = binaries/windows/pac_386
+        endif
+    endif
```

Comment:
The whole bunch of changes above is to generate different binaries on 32bit and 64bit Windows. See https://blogs.msdn.microsoft.com/david.wang/2006/03/27/howto-detect-process-bitness/ for reference.